### ISSUE TITLE
Лишние запросы к БД для проверки роли

### ISSUE THREAD

**Issue by undermuz** [NailyaKhafaeva/rest-api-server#4]
https://github.com/NailyaKhafaeva/rest-api-server/blob/main/src/auth/auth.service.ts#L45

В токене следует хранить данные о роли, чтобы не делать лишние запросы к БД при каждом обращении к контроллерам

**Comment by NailyaKhafaeva** [comment]
@undermuz Да, можно хранить в токене информацию о роли пользователя, но это не очень безопасно. Также, если мы будем хранить роль в токене, и при этом роль поменяется, то в токене она не обновится.

**Comment by undermuz** [comment]
@NailyaKhafaeva 
Почему не безопасно? роль это не секретные данные
Роль меняется редко, обычно такие фундаментальные изменения в данных пользователя требуют обновления токена - это учитывается в кейсах использования

**Comment by NailyaKhafaeva** [comment]
@undermuz  Хранение роли пользователя в токене зависит от установленных рамок конфиденциальности информации, если в данном случае роль — публичные данные, то я исправлю. Спасибо!
